video/vdu.h

```diff
@@ -194,25 +194,25 @@ void VDUStreamProcessor::vdu_print(char c, bool usePeek) {
 	s += c;
 	// gather our string for printing
 	if (usePeek) {
-		auto limit = 39;
+		auto limit = 15;
 		while (--limit) {
 			if (!byteAvailable()) {
 				break;
 			}
-			auto next = peekByte_t(FAST_COMMS_TIMEOUT);
+			auto next = inputStream->peek();
 			if (next == 27) {
-				readByte_t();
+				inputStream->read();
 				if (consoleMode) {
 					DBGSerial.write(next);
 				}
-				next = readByte_t(FAST_COMMS_TIMEOUT);
+				next = readByte_t();
```

Comment:
this call removes the timeout - we're trying to catch a character after an escape here to ensure it gets printed (for use printing characters 0-31 and 127) so we need to ensure that we wait around long enough for the character to appear.  if we miss the character and break at this point then there is a chance that the character will come and get misinterpreted as a command byte